test/api/v3/integration/groups/POST-groups_groupId_leave.js

```diff
@@ -93,7 +93,7 @@ describe('POST /groups/:groupId/leave', () => {
 
           let userWithChallengeTasks = await leader.get('/user');
 
-          expect(userWithChallengeTasks.challenges).to.not.include(challenge._id);
+          expect(userWithChallengeTasks.challenges).to.include(challenge._id);
```

Comment:
I think the it block of this test should specify that the user remains in the challenge.